libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/ImageEditor.kt

```diff
@@ -46,20 +47,36 @@ class ImageEditor private constructor(
         loadIntoFileWithResultListener.invoke(imageUrl, listener)
     }
 
+    fun trackEventWithActionData(data: Pair<TrackableEvent, TrackableAction?>) {
+        trackData(data)
```

Comment:
How would you feel about using a sealed class instead of two enums? Using two enums is in general fine if all the combinations are valid. However, the current interface supports for example sending `TrackableAction-CROP with TrackableEvent-MEDIA_EDITOR_SHOWN` which AFAICT doesn't make sense. Wdyt?